src/main/java/core/selenium/TrabajopolisConfig.java

```diff
@@ -68,9 +69,12 @@ public Properties config() {
             urlLogin = properties.getProperty("urlLogin");
             urlProfile = properties.getProperty("urlProfile");
             urlBase = properties.getProperty("urlBase");
+            urlCurriculum = properties.getProperty("urlCurriculum");
+            urlFillCurriculum = properties.getProperty("urlFillCurriculum");
         } catch (FileNotFoundException e) {
             e.printStackTrace();
             throw new NullPointerException("This file not exist!!");
+
         } catch (Exception e) {
             e.printStackTrace();
```

Comment:
Please add a throw Exception instead of printStackTrace.

Reply:
done